portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,101 @@
   width: 650px;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
 }
+
+.section-title {
+    font-family: Arial;
+    font-size: 14px;
+    font-weight: 600;
+    line-height: 14px;
+    padding-bottom: 1.5rem;
+    padding-top: 2.5rem;
+    text-align: center;
+}
+
+/* about me start */
+
+#about-me-container {
+    background-color:#537483;
+    padding-bottom: 2.5rem;
+}
+
+#about-me-content {
+    background-color: #ffffff;
```

Comment:
Use 3 character hexadecimal notation where possible: https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation

Apply throughout the file.

Reply:
I will change it. Thanks!